<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenFisca</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/OpenFisca22.png</normaloff>:/images/OpenFisca22.png</iconset>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actExportCsv"/>
    <addaction name="actExportPng"/>
    <addaction name="separator"/>
    <addaction name="actPreferences"/>
    <addaction name="separator"/>
    <addaction name="actFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actFileRefresh"/>
    <addaction name="separator"/>
    <addaction name="actCasType"/>
    <addaction name="actBareme"/>
    <addaction name="separator"/>
    <addaction name="actModeReforme"/>
   </widget>
   <widget class="QMenu" name="menu_dition">
    <property name="title">
     <string>Édition</string>
    </property>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="menu_dition"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actExportCsv"/>
   <addaction name="actExportPng"/>
   <addaction name="separator"/>
   <addaction name="actFileRefresh"/>
   <addaction name="separator"/>
   <addaction name="actCasType"/>
   <addaction name="actBareme"/>
   <addaction name="separator"/>
   <addaction name="actModeReforme"/>
   <addaction name="separator"/>
   <addaction name="actPreferences"/>
  </widget>
  <action name="actFileQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/process-stop.png</normaloff>:/images/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actFileRefresh">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Calculer</string>
   </property>
   <property name="toolTip">
    <string>Calculer (F9)</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actPreferences">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/preferences-desktop.png</normaloff>:/images/preferences-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Préférences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actExportCsv">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/document-save csv.png</normaloff>:/images/document-save csv.png</iconset>
   </property>
   <property name="text">
    <string>Exporter la table</string>
   </property>
   <property name="toolTip">
    <string>Exporter la table</string>
   </property>
  </action>
  <action name="actExportPng">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/document-save png.png</normaloff>:/images/document-save png.png</iconset>
   </property>
   <property name="text">
    <string>Exporter le graphique</string>
   </property>
   <property name="toolTip">
    <string>Exporter le graphique</string>
   </property>
  </action>
  <action name="actModeReforme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/comparison22.png</normaloff>:/images/comparison22.png</iconset>
   </property>
   <property name="text">
    <string>Réforme</string>
   </property>
   <property name="toolTip">
    <string>Différence entre la situation simulée et la situation actuelle</string>
   </property>
  </action>
  <action name="actCasType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/castype22.png</normaloff>:/images/castype22.png</iconset>
   </property>
   <property name="text">
    <string>Cas type</string>
   </property>
   <property name="toolTip">
    <string>Mode cas type</string>
   </property>
  </action>
  <action name="actBareme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/bareme22.png</normaloff>:/images/bareme22.png</iconset>
   </property>
   <property name="text">
    <string>Barème</string>
   </property>
   <property name="toolTip">
    <string>Mode Barème</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
